src/main/java/com/mycelium/spvmodule/guava/Bip44NotificationManager.kt

```diff
@@ -40,8 +40,8 @@ class Bip44NotificationManager {
         val connectivityNotificationEnabled = configuration.connectivityNotificationEnabled
 
         //We need to check fo 100 to prevent not partial sync on first run.
-        if (Bip44AccountIdleService.getSyncProgress() == 100F) {
-            Bip44AccountIdleService.getInstance()!!.stopForeground(false)
+        if (Bip44DownloadProgressTracker.getSyncProgress() == 100F) {
+            Bip44AccountIdleService.getInstance().stopForeground(false)
```

Comment:
starting the service, wait until it's started to then stop it looks wrong. Maybe add `getInstanceUnsafe()` for where getting null is actually ok. The `!!` was thus probably also wrong here and should have been `?.` to skip stopping if there is nothing to stop.